### ISSUE TITLE
error C2065: 'libnvidia_ml' when build

### ISSUE THREAD

**Issue by drr3d** [no-fee-ethereum-mining/nsfminer#127]
hi there..

just pulled up today and found some error when building from source..

is there anyone encounter this error 

> error C2065: 'libnvidia_ml'

 when building on Windows 10 pro 64bit..
currently using new driver version 461.40. and using visual studio 2017 CE

![err_nsfminer](https://user-images.githubusercontent.com/6335497/109374184-f8167f80-78e5-11eb-9fd4-33656c0e8641.PNG)

**Comment by danielocdh** [comment]
I had the same issue while trying to build earlier, I would guess it's a windows only issue. I don't really know much c++ so this might not be the right way to do it but I managed to compile and it worked for some testing, this is my git diff:

```
diff --git a/libhwmon/wrapnvml.cpp b/libhwmon/wrapnvml.cpp
index bd0668bee..d8aedf663 100644
--- a/libhwmon/wrapnvml.cpp
+++ b/libhwmon/wrapnvml.cpp
@@ -51,22 +51,26 @@ wrap_nvml_handle* wrap_nvml_create()
 #endif

     void* nvml_dll = nullptr;
-    char tmp[512] = libnvidia_ml;
+    //char tmp[512] = libnvidia_ml;

 #ifdef _WIN32
+    char tmp[512] = libnvidia_ml1;
     ExpandEnvironmentStringsA(libnvidia_ml1, tmp, sizeof(tmp));
     nvml_dll = wrap_dlopen(tmp);
     if (nvml_dll == nullptr)
     {
+        char tmp[512] = libnvidia_ml2;
         ExpandEnvironmentStringsA(libnvidia_ml2, tmp, sizeof(tmp));
         nvml_dll = wrap_dlopen(tmp);
         if (nvml_dll == nullptr)
         {
+            char tmp[512] = libnvidia_ml3;
             ExpandEnvironmentStringsA(libnvidia_ml3, tmp, sizeof(tmp));
             nvml_dll = wrap_dlopen(tmp);
         }
     }
 #else
+    char tmp[512] = libnvidia_ml;
     nvml_dll = wrap_dlopen(tmp);
 #endif
     if (nvml_dll == nullptr)
```

**Comment by drr3d** [comment]
hi @danielocdh 

thx for the reply..
i managed my self just change this following line:
before:
```
char tmp[512] = libnvidia_ml;
```
after:
```
char tmp[512] = libnvidia_ml3;
```

selecting `libnvidia_ml2` or `libnvidia_ml3` is worked for me too..as long as the library is there.

```
 I would guess it's a windows only issue
```
yap...i can agree with that

**Comment by jean-m-cyr** [comment]
Yes, windows only. Sorry about that! Fixed by commit 29b88bc
